build/ci/evergreen.yml

```diff
@@ -1928,6 +2014,188 @@ tasks:
           edition: org
           distro: debian10
           ext: deb
+  ################################################################
```

Comment:
let's use some new features and move this and the original each to new files https://github.com/evergreen-ci/evergreen/wiki/Project-Configuration-Files#include

Reply:
I will create a new yaml file containing all the funcs/steps/variants for releasing

Reply:
Updated